src/runtime_src/core/tools/xbutil2/SubCmdExamine.cpp

```diff
@@ -148,7 +148,7 @@ SubCmdExamine::execute(const SubCmdOptions& _options) const
 
   // Determine report level
   if (devices.size() == 0 && reportNames.size() == 0)
-      reportNames.push_back("host");
+    reportNames.push_back("host");
```

Comment:
warning: use emplace_back instead of push_back [hicpp-use-emplace]

```cpp
reportNames.push_back("host");
                ^
note: this fix will not be applied because it overlaps with another fix
```